backend/config/environments/test.rb

```diff
@@ -61,3 +61,6 @@
 
   config.active_job.queue_adapter = :test
 end
+
+# Set the default host and port to be the same as Action Mailer.
+Rails.application.default_url_options = Rails.application.config.action_mailer.default_url_options
```

Comment:
This is probably caused by https://github.com/Vizzuality/heco-invest/pull/168. The `default_url_options` is setup by `BACKEND_URL` which is parsed by `URI.parse` at first, which helps to extra `host`, `port` and `scheme` from it. I would check if your  `BACKEND_URL` variable at your local machine is parsable by `URI.parse`, maybe this will help ;).
By the way, @tsubik already mentioned this, but `BACKEND_URL` and `MAILER_HOST` are kind of duplicate, maybe we could keep just one -- vote to keep `BACKEND_URL`

Reply:
OK, thanks! Sorry I lost track of that